.travis.yml

```diff
@@ -15,10 +15,12 @@ before_install:
     - sudo pip install pyopenssl

```

Comment:
Why are `^M` (carriage return) characters everywhere in the file?
Strip them in vim using `:s/^M$//`